### PR TITLE
saving to local storage

### PR DESCRIPTION
**Pull request by GPain2007**
I changed the code it should store but getting errors?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
-<input id="'username" type="text" placeholder="username" /><br />
-<input id="'password" type="password" placeholder="password" /><br />
-<button onclick="storeData">SAve Info</button>
+<input id="username" type="text" placeholder="username" /><br />
+<input id="password" type="password" placeholder="password" /><br />
+<button onclick="storeData();">SAve Info</button>
```
Comment (GPain2007) [inline index.html:4]: this onClick should work correct?

File: lib/saveInfo.js [hunk 2]
```diff
@@ -1,18 +1,14 @@
-if (!localStorage.getItem("credentials")) {
-  localStorage.setItem("credentials", JSON.stringify([]));
-}
-
-const arr = JSON.parse(localStorage.getItem("credentials"));
-console.log(arr);
+function storeData() {
+  const newData = document.getElementById(".username").value;
```
Comment (GPain2007) [inline lib/saveInfo.js]: saying that the value is null but it should take in whatever I assign the data key to correct?

Comment (andrewvenson) [reply lib/saveInfo.js]: Your syntax is invalid. Remove the '.' from username

File: index.html (hunk 1 above)
Comment (andrewvenson) [reply index.html:4]: Remove the semicolon. You don't need that.

File: lib/saveInfo.js (hunk 2 above)
Comment (GPain2007) [reply lib/saveInfo.js]: Did that and coded out the password input because it was throwing an error. But I still can't get the code data to local storage.

**Pull request closed without merging**